timeline/plugins.py

```diff
@@ -114,3 +114,28 @@ class SiteAppPlugin(SiteAppPluginPoint):
 
     #: Required permission for displaying the app
     app_permission = 'timeline.view_site_timeline'
+
+
+class AdminSiteAppPlugin(SiteAppPluginPoint):
+    """Projectroles plugin for registering the app"""
+
+    #: Name (slug-safe, used in URLs)
+    name = 'admin_site'
```

Comment:
Plugin names are global, so we should be careful with these and follow a certain hierarchy and conventions. The other plugins within `timeline.plugins` are named as follows:

- `timeline`
- `timeline_backend`
- `timeline_site`

Hence the logical name would be `timeline_site_admin`.

Reply:
- [x] Changed